JSON.h

```diff
@@ -93,6 +93,29 @@ class JSON
         return std::get<T>(data[key]);
     }
 
+    /**
+     * If the given element is exists then gets it.
+     * If does not exists any element with the given key, then it will give back the given default element.
+     * @param key Map element's key.
+     * @param elseEl Other default element
+     * @return The searched element or a default element.
+     */
+    template <typename T>
+    T getOrElse(const std::string &key, T elseEl)
+    {
+        return keyExists(key) ? get<T>(key) : elseEl;
+    }
+
+    /**
+     * Checks the given key existance inside the map.
+     * @param key Map element key.
+     * @return Existance of the key.
+     */
+    bool keyExists(const std::string &key)
```

Comment:
Egy apróság: Új függvény helyett a már létező `count` is használható.

Reply:
Véleményem szerint jobban áttekinthetőbb a kód az új függvénnyel, de belsejét megváltoztathatom hogy a countot használja.

Reply:
Bár jobban megnézve elég a count is viszont szívem szerint átnevezném, mert engem is megzavart.

Reply:
Az áttekinthetőség miatt, a `count` az std része (https://www.cplusplus.com/reference/map/map/count/) is, így az osztály használata megegyezik a map használatával (ebben a funkcióban)

Reply:
És lehet a count függvény értéke nagyobb mint 1?

Reply:
> Because all elements in a map container are unique, the function can only return 1 (if the element is found) or zero (otherwise).
https://www.cplusplus.com/reference/map/map/count/

Reply:
Akkor jól gondoltam.